### PR TITLE
WCONPROD(TARGET) Tests 

### PR DESCRIPTION
**Pull request by OPMUSER**
The default tokens of 1* and '' should behave the same; however, the actual values used when WCONPROD(TARGET) is defaulted is variable, as outlined below:

If the well is not under group control, then WCONPROD(TARGET) is set to the first non-defaulted hydrocarbon rate (ORAT, GRAT, LIQ, RESV). If all rates are defaulted and BHP is entered, then the BHP is used. It is assumed that if the BHP is defaulted as well, and THP has been entered, then THP would be used (this configuration has not been tested). Finally, if all parameters are defaulted, then the default value of BHP (14.70 psia) will be used combined with BHP control. In all the above scenarios the well has been declared as open, that is WCONPROD(STATUS) is set to OPEN.

If, and only if, the well’s group parameters have been defined via the GCONPROD keyword, then WCONPROD(TARGET) is set equal to GRUP.

In summary, for wells belonging to groups, where GCONPROD has not been set or defined, then WCONPROD(TARGET) is defined as the first non-defaulted value on the WCONPROD keyword, except for WAT control mode. If all parameters are defaulted, then the default BHP value is used and the WCONPROD(TARGET) is set to BHP.

### CODE REVIEW

**Comment by OPMUSER** [comment]
Ready for review.

File: wconprod/WCONPROD-01.DATA [hunk 1]
```diff
@@ -0,0 +1,500 @@
...
+--
+-- --       WELL PRODUCTION WELL CONTROLS
+-- --
+-- -- WELL  OPEN/  CNTL   OIL    WAT    GAS   LIQ    RES    BHP   THP   VFP    VFP
+-- -- NAME  SHUT   MODE   RATE   RATE   RATE  RATE   RATE   PRES  PRES  TABLE  ALFQ
+-- WCONPROD
+-- OP01     OPEN   1*     20E3   1*     1*    1*     1*    1000.0                 /
+-- /
+--
+-- 456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012
```
Comment (alfbr) [inline wconprod/WCONPROD-01.DATA:56]: Is this line intentional?

**Comment by alfbr** [comment]
Generally speaking, this is a welcome addition and ready to go in. Checking possible run time issues can typically be done when the data sets are set up for integration testing. Please check the line I commented in-line, it seems present in all cases, is it really intentional? My only reservation is the change to the copyright header, why do you introduce a new license and a new layout for the copyright header?

**Comment by OPMUSER** [comment]
The line with the column numbering is intentional, but the copyright notice was not. I generated these cases to test the logic and then decided to submit the runs as test cases. I just used the wrong template for the copyright notice for the GitHub versions. Will corrected and re-submit.

**Comment by alfbr** [comment]
jenkins build this please
